src/components/Stake/CardAccordion.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonIcon,
+  Card,
+  GU,
+  IconUp,
+  IconDown,
+  RADIUS,
+  useTheme,
+} from '@aragon/ui'
+import { Spring, Transition, animated } from 'react-spring/renderprops'
+import styled from 'styled-components'
+
+const CardAccordion = React.memo(function CardAccordion({ card, expansion }) {
+  const [opened, setOpened] = useState(false)
+  const theme = useTheme()
+
+  const toggleButton = useCallback((index) => {
+    setOpened((opened) => !opened)
+  }, [])
+
+  return (
+    <div
+      css={`
+        position: relative;
+        margin-bottom: ${RADIUS}px;
+      `}
+    >
+      <OpenedSurfaceBorder opened={opened} />
+      <Card
+        css={`
+          width: 100%;
+          z-index: 2;
+        `}
+      >
+        <ToggleButton onClick={toggleButton} opened={opened} />
+        {card}
+      </Card>
+      <div
+        css={`
+          margin-top: ${-RADIUS}px;
+        `}
+      >
+        <Transition
+          native
+          items={opened}
+          from={{ height: 0 }}
+          enter={{ height: 'auto' }}
+          leave={{ height: 0 }}
+        >
+          {(show) =>
+            show &&
+            ((props) => (
+              <Expansion
+                css={`
+                  background: ${theme.surfaceUnder};
+                  overflow: hidden;
+                `}
+                style={props}
+              >
+                <div
+                  css={`
+                    width: 100%;
+                    overflow: hidden;
+
+                    padding: ${4 * GU}px ${3 * GU}px ${3 * GU}px;
+                    box-shadow: inset 0 ${RADIUS + 4}px 4px -4px rgba(0, 0, 0, 0.16);
+                  `}
+                >
+                  {expansion}
+                </div>
+              </Expansion>
+            ))
+          }
+        </Transition>
+      </div>
+    </div>
+  )
+})
+
+const Expansion = styled(animated.div)`
+  z-index: 1;
+`
```

Comment:
This is fine, but I think we've generally been moving away from separated styled components. What do you think about declaring a single `AnimatedDiv` and moving this inline?

```javascript
<AnimatedDiv style={props} css={`...customcss`}>
```